### DISCUSSION TITLE
Disabled prop for Switch

### DISCUSSION THREAD

**Discussion by u12206050** [tailwindlabs/headlessui#1169] [Ideas]
Hi Tailwind Team,

It seems like quite some of the other components have a disabled prop, but am missing it on the Switch component.

Would really like to have that option.

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

The Switch component by default renders a `button` which already has a `disabled` prop 👍 
E.g.: https://codesandbox.io/s/polished-waterfall-eumz2s?file=/src/App.js
